ustriage/ustriage.py

```diff
@@ -754,6 +756,13 @@ def launch():
                         dest='extended_format',
                         help='Do Display bugs in extended format which adds'
                              ' date-last-updated, importance and assignee')
+    parser.add_argument('-F', '--flag-recent',
+                        default=False,
```

Comment:
Even better: we can make `default=0` so `args.age` is always an int.

The logic also becomes very straightforward: "mark the bugs more recent than 0 days ago" => no bugs marked.

Reply:
Yes, right. Then I think it's fine as it is.